Sources/PackageModel/SwiftSDKBundle.swift

```diff
@@ -294,6 +295,20 @@ public struct SwiftSDKBundle {
         }
 
         try fileSystem.copy(from: unpackedBundlePath, to: installedBundlePath)
+
+        #if os(macOS)
+        // Remove the quarantine attribute from bundles downloaded manually in the browser.
+        let result = try Process.popen(
+            arguments: ["xattr", "-d", "-r", "-s", "com.apple.quarantine", installedBundlePath.pathString]
```

Comment:
Could we use `removexattr` (or similar) instead of shelling out?

Reply:
+1 if possible